ui/app/AppLayouts/Wallet/services/dapps/DappsConnectorSDK.qml

```diff
@@ -26,19 +26,291 @@ import "types"
 WalletConnectSDKBase {
     id: root
 
-    // Nim connector.controller instance
+    required property WalletConnectService wcService
+    required property var walletStore
+    required property DAppsStore store
+    required property int loginType
+
     property var controller
+    property var dappInfo: null
+    property var txArgs: null
     property bool sdkReady: true
     property bool active: true
-    required property WalletConnectService wcService
-    required property var walletStore
     property string requestId: ""
+    property alias requestsModel: requests
 
     projectId: ""
 
     implicitWidth: 1
     implicitHeight: 1
 
+    // TODO Refactor this code to avoid code duplication from Wallet Connect
```

Comment:
Could you add a refactor issue for 2.31 to get rid of all the duplications. Thanks

Reply:
Updated